src/ui/Views/Course/Index.cshtml

```diff
@@ -96,7 +96,7 @@
               @if(course.IsRunning()) {
                   <a href="@searchAndCompareUrlService.GetCoursePageUri(course.Provider.ProviderCode, course.CourseCode)">Yes - view online</a>
               } else {
-                <p class="govuk-body govuk-!-margin-0">@(course.IsNotRunning() ? "No" : "No – must be published on UCAS")</p>
```

Comment:
Can we keep the old wording for opted-out providers and the new wording for opted-in ones (steer from @fofr )

Reply:
This is because non-opted in providers still need to be signed posted to UCAS.